### PR TITLE
[CL][89149004] added event trigger on form load

### PR DESCRIPTION
**Pull request by cleathers**
[Story](https://www.pivotaltracker.com/story/show/89149004)

I need a way to know when the form has successfully loaded. Had tried using 'DOMSubtreeModified' in my ui component, but that won't work in IE8. Not sure if this is the best way to remedy that situation.

Related to this PR: https://github.com/rentpath/rent/pull/167

### CODE REVIEW

File: dist/jquery-moving-service.js [hunk 1]
```diff
@@ -80,6 +80,7 @@
           $(".moving_form", form_div).submit(submitLead);
           setup_bindings();
           form_div.show();
+          $(document).trigger('uiMovingFormLoaded');
```
Comment (c0) [inline dist/jquery-moving-service.js]: What do you think about naming this `uiMovingFormShown` or `uiMovingFormVisible`?

**Comment by c0** [comment]
Events are much, much better than `DOMSubtreeModified`. :+1:

File: dist/jquery-moving-service.js (hunk 1 above)
Comment (cleathers) [inline dist/jquery-moving-service.js]: Sounds good to me. I've changed the event name to uiMovingFormShown.

**Comment by c0** [comment]
:+1:

**Comment by cleathers** [comment]
Thanks @c0

**Pull request merged**